aspnetcore/blazor/hybrid/developer-tools.md

```diff
@@ -86,20 +93,42 @@ To use Safari developer tools with an iOS app:
 
 :::zone pivot="macos"
 
-Using browser developer tools with :::no-loc text="Mac Catalyst"::: apps isn't currently supported. You may wish to run the [iOS](?pivots=ios) application instead, as the behavior is similar between iOS and :::no-loc text="Mac Catalyst":::.
+<!-- On macOS, XML files use 4-space indents. Also, the PU uses 4-space indents in the .NET MAUI template file. -->
+
+Add the `com.apple.security.get-task-allow` key, of type `Boolean`, to the [entitlements file](/dotnet/maui/ios/entitlements) of the app for its debug build.
+
+To add an entitlements file with the `com.apple.security.get-task-allow` key, add the following XML file named `Entitlements.Debug.plist` to the `Platforms/MacCatalyst` folder of the project.
+
+`Platforms/MacCatalyst/Entitlements.Debug.plist`:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>com.apple.security.get-task-allow</key>
+    <true/>
+</dict>
+</plist>
+```
+
+To consume the entitlements file for debug builds on :::no-loc text="Mac Catalyst":::, add the following `<PropertyGroup>` node to the app's project file as a child of the `<Project>` node:
+
+```xml
+<PropertyGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'maccatalyst' and '$(Configuration)' == 'Debug'">
+    <CodeSignEntitlements>Platforms/MacCatalyst/Entitlements.Debug.plist</CodeSignEntitlements>
+</PropertyGroup>
+```
 
-<!--
 To use Safari developer tools with a macOS app:
 
 1. Open desktop Safari.
-1. Select the **Preferences** > **Advanced** > **Show Develop** menu command.
+1. Select the **Safari** > **Preferences** > **Advanced** > **Show Develop menu in the menu bar** checkbox.
 1. Run the .NET MAUI Blazor app in the macOS simulator.
```

Comment:
It's not running in a simulator. It's just running in macOS.

Reply:
This merely occurred from coping over content from the iOS content, where a simulator is run. I'll fix it up on the commit.